boards/rpi-pico/include/board.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         boards_rpi_pico
+ * @{
+ *
+ * @file
+ * @brief           Board specific definitions for the Raspberry Pi Pico
+ *
+ * @author          Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LED configuration
+ * @{
+ */
+#define LED0_PIN        GPIO_PIN(0, 25)
+#define LED0_ON         do {SIO->GPIO_OUT_SET.reg = 1UL << 25;} while(0)
```

Comment:
An earlier version of the driver actually did this, but I decided that this is not worth the trouble it causes in the initialization code. After all, a bit-wise left shift is a simple operation, the inverse is (especially in absence of CLZ on Cortex M0+) not.

Also: The CPU cycle safed is not going to matter much, since the function call for `gpio_set()` etc. is going to be the main contributor to CPU time. If you use LTO to avoid this, then the compiler will also (in most case) be able to do the shift at compile time. So there isn't anything to be win.

IMO, if we want a fast GPIO API, we should go in the direction @gschorcht is heading with the new GPIO API.